document_ai_warehouse_processing_python/requirements.txt

```diff
@@ -3,5 +3,4 @@ google-cloud-core==2.3.2
 google-cloud-documentai==2.4.1
```

Comment:
Minor: The latest version is `2.6.0` (Note: The library versions will be auto-updated in this repo)